automation/DeployToHPVS.bashlib

```diff
@@ -767,13 +767,13 @@ Passphrase: ${USERINPUT}
     #--------------------------------------------------------------------------#
     # Config variable: resource_group                                          #
     #--------------------------------------------------------------------------#
-    prompt_user "IBM Cloud Resource Group" true "default"
+    prompt_user "ID of IBM Cloud Resource Group to use for your deployment" true "default"
```

Comment:
I would add this link here, saying "you can find out more information about resource groups here https://cloud.ibm.com/docs/account?topic=account-rgs. "  And also say if you are unsure, leave the setting as default